package.json

```diff
@@ -53,10 +53,10 @@
     "@nestjs/core": "^5.1.0",
     "@nestjs/testing": "^5.1.0",
     "@types/dotenv": "^6.1.0",
-    "@types/glob": "^5.0.35",
-    "@types/jest": "^23.3.0",
+    "@types/glob": "^7.1.1",
+    "@types/jest": "^24.0.5",
     "coveralls": "^3.0.2",
-    "jest": "^23.3.0",
+    "jest": "^24.1.0",
     "prettier": "^1.13.7",
     "ts-jest": "^23.0.0",
```

Comment:
`ts-jest` should probably be the same major  version as `jest` (e.g. `24.X.X`)

Reply:
Upgrading `ts-jest` will also fix a lot of vulnerabilities in deps (e.g. `yarn audit`)